app/templates/people/person/positions/minister/edit.hbs

```diff
@@ -0,0 +1,259 @@
+                            this.model.minister.contacts.firstObject.contactAddress.province
+                            )
+                        }}
+                        />
+                    </:content>
+                    </Item>
+                </:left>
+                <:right as |Item|>
+                    <Item @labelFor="contact-email">
+                    <:label>E-mail</:label>
+                    <:content>
+                        <AuInput
+                        @width="block"
+                        @value={{this.model.minister.contacts.firstObject.email}}
+                        id="contact-email"
+                        />
+                    </:content>
+                    </Item>
+                    <Item @labelFor="contact-telephone">
+                    <:label>Telefoonnummer</:label>
+                    <:content>
+                        <AuInput
+                        @width="block"
+                        @value={{this.model.minister.contacts.firstObject.telephone}}
+                        id="contact-telephone"
+                        />
+                    </:content>
+                    </Item>
+                </:right>
+                </Card.Columns>
+            </:card>
+            </EditCard>
+
+            <EditCard>
+                <:title>Aanstelling</:title>
+                <:card as |Card|>
+                <Card.Columns>
+                    <:left as |Item|>
+                    <Item @labelFor="start-date">
+                        <:label>Startdatum</:label>
+                        <:content>
+                        <Datepicker
+                            @id="start-date"
+                            @onChange={{fn (mut this.model.minister.agentStartDate)}}
+                            @value={{this.model.minister.agentStartDate}}
+                        />
+                        </:content>
+                    </Item>
+                    </:left>
+                    <:right as |Item|>
+                    <Item @labelFor="end-date">
+                        <:label>Einddatum</:label>
+                        <:content>
+                        <Datepicker
+                            @disabled={{this.isCurrentPositionCheck}}
+                            @id="end-date"
+                            @onChange={{fn (mut this.model.minister.agentEndDate)}}
+                            @value={{this.model.minister.agentEndDate}}
+                        />
+                        </:content>
+                    </Item>
+                    <Item @labelFor="end-date">
+                        <:label></:label>
+                        <:content>
+                        <AuControlCheckbox
+                            @checked={{this.isCurrentPositionCheck}}
+                            @label="Huidige positie"
+                            @identifier="current-position"
+                            @onChange={{fn (mut this.isCurrentPositionCheck)}}
```

Comment:
I think we need to create an action for this. If the user selects a date, and then checks this check box, the date is kept. So if they then save, the endDate will still be saved, even though they checked the checkmark. So we either need to clear the endDate when they check the box, or we clear it when saving and the checkbox was checked.

I think the clear on check version is more intuitive since that provides visual feedback to the user.

I know it works like this in the creation page, but we can update that later.